index.html

```diff
@@ -68,6 +71,64 @@
             </nav>
         </div>
     </div>
-
+    <main class="page-content">
+        <div class="container">
+            <section class="promo-slider">
+                <div class="slider">
+                    <div class="slider__item slider__item--1">
+                        <div class="slider__img-wrap slider__img-wrap--01"></div>
+                        <div class="slider__descr-wrap slider__descr-wrap--1">
+                            <b>Делай селфи,<br>как Бен Стиллер!</b>
+                            <p>Самая длинная палка для селфи доступна в нашем магазине. Восемь (Восемь, Карл!) метров
+                                длиной и весом всего 5 килограмм.</p>
+                            <a class="btn" href="#">Подробнее</a>
+                            <div class="slider__advantages  slider__advantages--1">
```

Comment:
bad solution at all
example > 
slider__advantages-col>slider__advantages-value+slider__advantages-name 
something like that
refactor it please

Reply:
and dont forget to remove classes like that slider__advantages--1